src/Network/Network.Test/ScenarioTests/VirtualNetworkTests.ps1

```diff
@@ -165,12 +165,14 @@ function Test-subnetCRUD
 
         # Get subnet
         $subnet2 = Get-AzvirtualNetwork -Name $vnetName -ResourceGroupName $rgname | Get-AzVirtualNetworkSubnetConfig -Name $subnet2Name
+        $subnet2ById = Get-AzVirtualNetworkSubnetConfig -ResourceId $$subnet2.Id
```

Comment:
@lwajswaj nit: I'm not sure if there's any issue, but just to be safe, would you mind removing the extra `$` character in front of `subnet2.Id`?

Reply:
Done